molecule/driver/delegated.py

```diff
@@ -51,6 +51,14 @@ class Delegated(base.Base):
           port: ssh_port_as_string
           user: ssh_user
 
+        - address: winrm_endpoint
+          instance: instance_name
+          port: winrm_port
+          user: winrm_user
+          connection: 'winrm'
+    This article covers how to configure and use WinRM with Ansible:
```

Comment:
Does this render correctly without a newline between the code block and the paragraph?

Reply:
Fixed